app/views/npq_separation/admin/dashboards/summary/show.html.erb

```diff
@@ -1,20 +1,41 @@
 <h1 class="govuk-heading-l">Summary</h1>
 
 <%=
-  govuk_summary_list do |sl|
-    sl.with_row do|r|
-      r.with_key(text: "Applications this week")
-      r.with_value(text: @applications_this_week)
+  govuk_table do |table|
```

Comment:
since we're duplicating the tables here, is it worth moving those to a view component with the tally method where we can test it?